docs/AboutUs.md

```diff
@@ -4,49 +4,49 @@ We are a team based in the [School of Computing, National University of Singapor
 
 ## Project Team
 
-#### [Damith C. Rajapakse](http://www.comp.nus.edu.sg/~damithch) <br>
-<img src="images/DamithRajapakse.jpg" width="150"><br>
-**Role**: Project Advisor
+#### [Verbena Ong](http://github.com/Verbena) 
+<img src="images/Verbena.jpg" width="150"><br>
+**Role**: Team Leader<br>  
+Responsibilities: Project coordination, Integration
```

Comment:
you need to indicate who is in charge of which component of the project, like UI, Logic etc.,